code/game/gamemodes/roguetown/roguetown.dm

```diff
@@ -102,14 +102,14 @@ GLOBAL_LIST_INIT(roguegamemodes, list(
 		return TRUE*/
 
 /datum/game_mode/chaosmode/proc/check_for_lord(forced = FALSE)
-	if(!forced && world.time < next_check_lord)
```

Comment:
... you do realize you tried this while the pr that completely removes gamemodes is in yea